src/static/js/modules/util/web-storage-proxy.js

```diff
@@ -11,90 +11,107 @@
    for local storage (note omitted 'true' value).
    Note: values stored in local storage are not accessible from session storage
    and vice versa. They both work on different objects within the browser.
-   ========================================================================== */
+   ========================================================================= */
 
 'use strict';
 
-// Default session type.
-// true = sessionStorage (non-persistent).
-// false = localStorage (persistent).
-var _sessionOnly = false;
+// Default storage type.
+var _storage = window.sessionStorage;
 
 /**
  * Set an item value specified by the key in web storage.
  * @param {string} key The key for the value.
  * @param {string} value The value to store.
- * @param {boolean} sessionOnly (Optional)
- *   Use non-persistent storage (true) or persistent storage (false).
+ * @param {object} storage (Optional)
+ *   Use non-persistent storage (sessionStorage)
+ *   or persistent storage (localStorage).
```

Comment:
@anselmbradford 
I should also point out, you can pass whatever storage you want, it doesn't even need to be the window's local or session storage. You could use a storage object created within the project if you wanted to (which would amount to just the single page load, but still, it's possible).